ember/ember.d.ts

```diff
@@ -1765,9 +1788,43 @@ declare module Ember {
         toArray(): any[];
     }
     module RSVP {
+        interface IPromiseFunction {
+            (value: any):void;
+        }
+
+        interface IPromiseResolverFunction {
+            (resolve: IPromiseFunction, reject: IPromiseFunction): void;

```

Comment:
If that's not good enough for you, I'm OK with `ResolveCallback` and `RejectCallback` too.